### DISCUSSION TITLE
Is this on purpose?

### DISCUSSION THREAD

**Discussion by scc02** [catppuccin/nvim#557] [Q&A] [answered]
<img width="1318" alt="image" src="https://github.com/catppuccin/nvim/assets/22486446/f31b94a4-3821-47cc-b852-5b57581180b9">
Same text, differenct color, The filetype of left one is `javascriptreact',right one is 'typescriptreact',Is this on purpose?

**Comment by scc02** [comment, marked as answer]
Ok,It's on purpose
<img width="1248" alt="image" src="https://github.com/catppuccin/nvim/assets/22486446/4743a134-d07f-43e5-9139-bdf004ba03d3">

**Reply by michaeldebetaz** [reply]
Same surprise for me. I don't understand why the highlighting differs between jsx and tsx 🤔 And the jsx makes definitely more sense...
